components/eamxx/src/share/io/scorpio_output.cpp

```diff
@@ -848,31 +848,41 @@ void AtmosphereOutput::register_views()
 /* ---------------------------------------------------------- */
 void AtmosphereOutput::set_avg_cnt_tracking(const std::string& name, const std::string& avg_cnt_suffix, const FieldLayout& layout)
 {
-    // Make sure this field "name" hasn't already been regsitered with avg_cnt tracking.
-    // Note, we check this because some diagnostics need to have their own tracking which
-    // is created at the 'create_diagnostics' function.
-    if (m_field_to_avg_cnt_map.count(name)>0) {
-      return;
+  // Make sure this field "name" hasn't already been regsitered with avg_cnt tracking.
```

Comment:
There is a typo in "registered" which is held over from when I put this comment in the code.

Reply:
We can put it in one of our next PRs.